automl/beta/video_object_tracking_create_dataset_test.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+from google.cloud import automl_v1beta1 as automl
+import pytest
+
+import video_object_tracking_create_dataset
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+pytest.DATASET_ID = None
+
+
+@pytest.fixture(scope="function", autouse=True)
+def teardown():
+    yield
```

Comment:
Is it possible to delete by dataset name? It would be preferable if the fixture would return the name w/ uuid and then clean it up afterwards. 

(I feel very iffy about adding an attribute to a third-party module)

Reply:
No, you have to use the ID which is created by the API. Yea, I've spent time looking for a better solution, I just haven't found one yet, beside using globals. 

One solution was to use a class based path and just set that. I'm open to ideas.

Reply:
I think it's better to just have a regular global over assigning it to third party module - otherwise your solution seems good given the constraints.